Python/README.md

```diff
@@ -1,3 +1,91 @@
+
+<p align="center">
```

Comment:
why edited complete readme and not just added your line of edit?

Reply:
I was just adding my line only, but I don't really know how this whole readme file got edited. What should I do?

Reply:
try comparing with the previous version and check all edits.